pkg/configmap/configmap_test.go

```diff
@@ -196,20 +197,34 @@ func TestDelete(t *testing.T) {
 	}
 }
 
-func TestWithOptions(t *testing.T) {
-	testBuilder := buildTestBuilderWithFakeObjects([]runtime.Object{})
-
-	testBuilder.WithOptions(func(builder *Builder) (*Builder, error) {
-		return builder, nil
-	})
-
-	assert.Equal(t, "", testBuilder.errorMsg)
+func TestUpdate(t *testing.T) {
+	testCases := []struct {
+		testCM        *Builder
+		testData      map[string]string
+		expectedError error
+	}{
+		{
+			testCM:        buildTestBuilderWithFakeObjects([]runtime.Object{}),
```

Comment:
Please add a test where a configmap resource is added to runtime objects

Reply:
@trewest this is secret, not configmap.
do you ask me to add runtime test to the "With..." method?

Reply:
I think it is an issue with either the title or contents then? The title says secret pkg but it looks like you updated configmap files